buildroot-external/patches/occu/0085-WebUI-AddonPopupScrollable.patch

```diff
@@ -0,0 +1,18 @@
+--- occu/WebUI/www/config/cp_software.cgi.orig
++++ occu/WebUI/www/config/cp_software.cgi
+@@ -243,6 +243,7 @@
+     puts "\${dialogSettingsExtraSoftwareTitle}"
+   }
+   division {class="CLASS21406"} {
++    puts "<div style='height:400px;width:100%;overflow:scroll;'>"

```

Comment:
Sehe ich das richtig, das hier bewusst das `height:400px` gesetzt wurde weil ein dynamischer wert in % nicht möglich ist? Kann man nicht irgendwie herausbekommen wie groß das Browserfenster zur zeit des öffnen des Popups gerade ist und dann den wert an der Stelle entsprechend anpassen?

Wenn nein, wäre die Frage ob hier der Wert von `400px` wirklich ein Wert ist der für die meisten Nutzer so akzeptabel ist. Und wie verhält sich das Popup wenn man ein Browserfenster hat das kleiner als 400px ist und das Popup da schon nicht mehr reinpasst?

Reply:
> Sehe ich das richtig, das hier bewusst das height:400px gesetzt wurde weil ein dynamischer wert in % nicht möglich ist? 

Genau, aufgrund der Tatsache, dass der Tabelleninhalt (Zeilen) dynamisch gebaut wird, hat eine prozentuale Höhe keine Funktion.

> Kann man nicht irgendwie herausbekommen wie groß das Browserfenster zur zeit des öffnen des Popups gerade ist und dann den wert an der Stelle entsprechend anpassen?

Wäre besser, ja. Muss ich mal googlen...

> wie verhält sich das Popup wenn man ein Browserfenster hat das kleiner als 400px ist und das Popup da schon nicht mehr reinpasst?

Dann wird die ganze Seite _von sich aus?_ scrollable inkl. dem darin befindlichen Popup.

Reply:
> Genau, aufgrund der Tatsache, dass der Tabelleninhalt (Zeilen) dynamisch gebaut wird, hat eine prozentuale Höhe keine Funktion.

Gibt es keine Möglichkeit hier den prozentualen Anteil der Browserfenstergröße bzw. des äußeren div in dem das Popup landet zu verwenden?

> > Kann man nicht irgendwie herausbekommen wie groß das Browserfenster zur zeit des öffnen des Popups gerade ist und dann den wert an der Stelle entsprechend anpassen?
> 
> Wäre besser, ja. Muss ich mal googlen...

Das wäre wirklich super wenn wir da ne möglichkeit finden könnten zu sagen "nimm sowas wie 80% des browserfensters" oder so...

> > wie verhält sich das Popup wenn man ein Browserfenster hat das kleiner als 400px ist und das Popup da schon nicht mehr reinpasst?
> 
> Dann wird die ganze Seite _von sich aus?_ scrollable inkl. dem darin befindlichen Popup.

Ok, das ergibt Sinn und ist so OK.

Reply:
Mit  af946a3 wird nun 50% der Fensterhöhe verwendet jedoch max. 400px. (ohne die max-Grenze sieht es nicht schön aus)